--- development-observed.ckdb
+++ development-proposed.ckdb
@@ -1,4 +1,20 @@
 DEFINE SCHEMA
+
+    RECORD TYPE MirrorRecord (
+        "___createTime" TIMESTAMP,
+        "___createdBy"  REFERENCE,
+        "___etag"       STRING,
+        "___modTime"    TIMESTAMP,
+        "___modifiedBy" REFERENCE,
+        "___recordID"   REFERENCE,
+        modifiedAt      TIMESTAMP,
+        payload         BYTES,
+        appendOnly      INT64,
+        hubOwnerID      STRING,
+        GRANT WRITE TO "_creator",
+        GRANT CREATE TO "_icloud",
+        GRANT READ TO "_creator"
+    );
 
     RECORD TYPE KithDocument (
         "___createTime" TIMESTAMP,
